properties-files/http.properties

```diff
@@ -29,3 +29,7 @@ HTTP_TEST_SMOKE_TEST=false
 HTTP_TEST_NAMESPACE_CLEANUP=true
 HTTP_TEST_STRESS_CONTAINER_IMAGE=quay.io/openshift-scale/http-stress
 HTTP_TEST_SERVER_CONTAINER_IMAGE=quay.io/openshift-scale/nginx
+ES_SERVER=
+ES_PORT=80
```

Comment:
Minor nit but any reason we are defining an ES port number default even though there is no server default?